flink-runtime-web/web-dashboard/web/css/vendor.css

```diff
@@ -1,96 +1,96 @@
 /* nvd3 version 1.8.4 (https://github.com/novus/nvd3) 2016-07-03 */
-.nvd3 .nv-axis {

```

Comment:
it would be nice if you could revert all these changes; they are mostly related to line-endings.